ipynb/tests/Generic_EAS_Tests.ipynb

```diff
@@ -0,0 +1,585 @@
6005]]}, \"sshd-3280\": {\"1\": [[7.0611420000059297, 7.0620120000094175]], \"2\": [[7.0621100000134902, 7.0646300000080373]]}, \"shutils-3266\": {\"0\": [[2.0260590000107186, 2.0266460000129882]], \"3\": [[2.0258130000001984, 2.0259710000100313]]}, \"shutils-3267\": {\"2\": [[2.0279840000002878, 2.0281820000091102]], \"4\": [[2.028269000002183, 2.0291630000137957]]}, \"shutils-3260\": {\"2\": [[2.0152040000102716, 2.0153040000004694]], \"4\": [[2.015454000007594, 2.016354000006686]]}, \"shutils-3261\": {\"2\": [[2.0178410000080476, 2.0180320000072243]], \"4\": [[2.0180510000063805, 2.0189340000069933]]}, \"shutils-3262\": {\"2\": [[2.019244000010076, 2.0193610000133049]], \"5\": [[2.0194570000021486, 2.0201090000045951]]}, \"shutils-3263\": {\"4\": [[2.0214660000056028, 2.0223270000133198]], \"5\": [[2.0210440000082599, 2.0213780000049155]]}, \"sh-3249\": {\"2\": [[8.5000006947666407e-05, 0.00027100001170765609]]}, \"kworker/1:0-889\": {\"1\": [[0.21906400000443682, 0.21922800000174902], [1.1871280000050319, 1.1871900000114692], [1.2190560000017285, 1.2192170000053011], [2.2191010000096867, 2.2192660000000615], [3.2191320000129053, 3.2193490000063321], [4.2191250000032596, 4.219293000001926], [5.2191280000115512, 5.2193840000109049], [6.2190000000118744, 6.2191680000105407], [7.2191190000012284, 7.2193460000125924], [8.2191290000046138, 8.2192980000108946]]}, \"sshd-3275\": {\"0\": [[6.8632890000008047, 6.8634330000058981], [6.865804000000935, 6.8710790000041015], [6.879865000009886, 6.880410000012489], [6.8806640000111656, 6.8808850000059465], [6.8812630000029458, 6.900619000007282], [6.9008230000035837, 6.9009030000015628], [6.9016030000057071, 6.9024380000046222], [6.9026290000037989, 6.9027170000044862]], \"1\": [[6.7724900000030175, 6.7728860000061104], [6.8712400000076741, 6.8794780000025639], [6.9073580000112997, 6.907457000008435], [6.9077930000057677, 6.9078500000032363], [6.907881000006455, 6.9084830000065267], [6.9091150000022026, 6.9091620000108378], [6.9093140000040876, 6.9095340000058059], [6.9097400000027847, 6.9104690000094706]], \"2\": [[6.7729960000142455, 6.7788480000017444], [6.7816750000056345, 6.7918920000083745], [6.7936480000062147, 6.7936680000129854], [6.793736000006902, 6.8166610000043875], [6.8243340000044554, 6.824374000003445], [6.9012460000085412, 6.9013400000112597]]}, \"sshd-3277\": {\"1\": [[6.9033190000045579, 6.9057820000016363]], \"2\": [[6.9023140000062995, 6.9032010000082664]]}, \"kworker/u12:2-3279\": {\"2\": [[6.9329410000063945, 6.9329740000102902]], \"4\": [[6.9330630000040401, 6.9348960000061197]]}, \"kworker/u12:2-3276\": {\"1\": [[6.7792940000072122, 6.7793430000019725]], \"3\": [[6.7795090000145137, 6.7814940000098431]]}, \"sshd-3278\": {\"0\": [[7.0153290000016568, 7.015495000014198], [7.0179130000033183, 7.0384260000137147], [7.0387300000002142, 7.0392610000126297], [7.0395130000106292, 7.0397250000096392], [7.0401260000071488, 7.0594920000003185], [7.0596900000091409, 7.0597150000103284], [7.0604
```

Comment:
.. is higher than the **ideal energy plot**.. 
Although a reword of it might work: "Where the values for the observed energy are higher than the ideal energy values..". The same for the following sentence. It's strange to say "where the plot is higher". Maybe a better wording would be "where the plot line is higher.. ".